lldb/include/lldb/Target/Platform.h

```diff
@@ -1004,10 +1004,7 @@ class Platform : public PluginInterface {
 
   virtual const char *GetCacheHostname();
 
-  virtual Status
-  ResolveRemoteExecutable(const ModuleSpec &module_spec,
-                          lldb::ModuleSP &exe_module_sp,
-                          const FileSpecList *module_search_paths_ptr);
+  /// The method is virtual for mocking in the unit tests.
```

Comment:
Which function is this comment referring to (if I'm reading the diff right, it's just floating there)?

Reply:
This comment is outdated. It was for `ResolveRemoteExecutable` which was virtual but only overridden in the unit test. I'll remove it.